lib/internal/webstreams/queuingstrategies.js

```diff
@@ -56,14 +56,20 @@ const byteSizeFunction = ObjectDefineProperty(
 );
 const countSizeFunction = ObjectDefineProperty(() => 1, 'name', nameDescriptor);
 
+const getSymbolToStringTagProperty = (value) => {
```

Comment:
If we are no longer using a getter, I suggest we use a different name. I'm not sure using a helper function is really useful in this case btw.
```suggestion
const nonWritablePropertyDescriptor = (value) => {
```

Reply:
added `get` prefix and made the function to `getNonWritablePropertyDescriptor`